src/common/runtime/cmdline.ts

```diff
@@ -16,6 +17,7 @@ function usage(rc: number): never {
   console.log(`  tools/run_${sys.type} [OPTIONS...] QUERIES...`);
   console.log(`  tools/run_${sys.type} 'unittests:*' 'webgpu:buffers,*'`);
   console.log('Options:');
+  console.log('  --colors             Enable ANSI colors in output.');
```

Comment:
Can we invert this to `--no-colors`? I'm going to forget to type it more on the command line then you would in a script I think?

Reply:
If you're using the `run-cts` script, then this will default enable it based on terminal detection logic.

Reply:
That works.